src/Microsoft.Sbom.Api/Executors/LicenseInformationService.cs

```diff
@@ -30,6 +29,11 @@ public LicenseInformationService(ILogger log, IRecorder recorder, HttpClient htt
     }
 
     public async Task<List<string>> FetchLicenseInformationFromAPI(List<string> listOfComponentsForApi)
+    {
+        return await FetchLicenseInformationFromAPI(listOfComponentsForApi, 30);
```

Comment:
Rather than a magic number, let's use a constant. Something like `DefaultClientTimeoutSeconds`, which could replace the previous `ClientTimeoutSeconds`

Reply:
I agree, I've updated LicenseInformationService and LicenseInformationFetcher to use a constant instead.